src/components/code-editor/code-editor.tsx

```
@@ -245,8 +293,40 @@
-
-        return <div class={classList} />;
+        return [
+            this.expandable && !this.dialogOpen && (
+                <limel-icon-button
+                    class="code-fullscreen-button code-fullscreen-button-focus"
+                    label="Expand"
+                    icon="expand"
+                    onClick={this.handleClickExpand}
+                    elevated={true}
+                    aria-label="Expand code editor"
+                />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)